.pipelines/templates/build-pipelines.yml

```diff
@@ -50,11 +50,14 @@ steps:
     echo "##vso[task.setvariable variable=\$id]$schemaId"
   displayName: Set dab version
 
+# The .NET CLI commands in proceeding tasks use the .NET SDK version specified here.
+# Per Microsoft Learn Docs, "Selecting the .NET SDK version is independent from 
```

Comment:
What does "selecting the version" mean here?

Reply:
clarifying with comment. "selecting the version" means installing a specific SDK version as done in this task.
```csharp
# The .NET CLI commands in proceeding tasks use the .NET SDK version specified ("selected") here.
# Per Microsoft Learn Docs, "Selecting the .NET SDK version is independent from 
# specifying the runtime version a project targets."
```